docs/openstack/nova_adoption.md

```diff
@@ -0,0 +1,176 @@
+  ```
+
+* Wait for Nova control plane services' CRs to become ready:
+
+  ```bash
+  oc get nova --field-selector metadata.name=nova-api -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaapis --field-selector metadata.name=nova-api -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novacells --field-selector metadata.name=nova-cell0 -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novacells --field-selector metadata.name=nova-cell1 -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaconductors --field-selector metadata.name=nova-cell0-conductor -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaconductors --field-selector metadata.name=nova-cell1-conductor -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novametadata --field-selector metadata.name=nova-metadata -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novanovncproxies --field-selector metadata.name=nova-cell1-novncproxy -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  oc get novaschedulers --field-selector metadata.name=nova-scheduler -o jsonpath='{.items[0].status.conditions}' \
+  | jq -e '.[]|select(.type=="Ready" and .status=="True")'
+  ```
+
+  The local Conductor services will be started for each cell, while the superconductor runs in `cell0`.
+  Note that ``disable_compute_service_check_for_ffu`` is mandatory for all imported Nova services, until
+  the [external dataplane imported](edpm_adoption.md), and until Nova Compute services fast-forward upgraded.
+
+## Post-checks
+
+* Check that Nova endpoints are defined and pointing to the
+  podified FQDNs and that Nova API responds.
+
+  ```bash
+  openstack endpoint list | grep nova
+  openstack server list
+  ```
+
+Compare the following outputs with the topology specific configuration
+[collected earlier](pull_openstack_configuration.md#get-services-topology-specific-configuration):
+
+* Query the superconductor for cell1 existance:
+
+  ```bash
+  oc rsh nova-cell0-conductor-0 nova-manage cell_v2 list_cells | grep -F '| cell1 |'
+  ```
+
+  The expected changes to happen:
+  * cell1's `nova` DB and user name become `nova_cell1`.
+  * Default cell is renamed to `cell1` (in a multi-cell setup, it should become indexed as the last cell instead).
+  * RabbitMQ transport URL no longer uses `guest`.
+
+* Verify no Nova compute dataplane disruptions during the adoption/upgrade process:
+
+  ```bash
+  $CONTROLLER_SSH sudo podman exec -it libvirt_virtqemud virsh list --all | grep 'instance-00000001   running'
```

Comment:
This cannot be executed as some of the pre steps actually stopped the container.

Reply:
I used `$CONTROLLER1_SSH ps auxf | grep guest=instance-00000001` instead to check that the domain is still running

Reply:
good point, this should go to https://github.com/openstack-k8s-operators/data-plane-adoption/blob/9844aa6e67b34e3dbffe3734897722619619b260/docs/openstack/pull_openstack_configuration.md#L49

Reply:
wait, you are confused it with EDPM service https://github.com/openstack-k8s-operators/edpm-ansible/blob/800dc2eb14cd253ab029286ce3104088d91c88fe/roles/edpm_libvirt/defaults/main.yml#L33 which should be up and running, but probably not yet - but after EDPM adoption completed

Reply:
Done https://github.com/openstack-k8s-operators/data-plane-adoption/compare/9844aa6e67b34e3dbffe3734897722619619b260..762b0ea97b0915fb3dbef78ff96e8570e5636b33